lib/std/Build.zig

```diff
@@ -1881,6 +1904,11 @@ pub const LazyPath = union(enum) {
     /// Use of this tag indicates a dependency on the host system.
     cwd_relative: []const u8,
 
+    dependency: struct {
```

Comment:
Nitpick, but I don't love this name: I think something like `package_file` would be better

Reply:
I don't love the name either, but think it's the most suitable and consistent one with the rest of that part of the build system. `package_file` doesn't really conform with the other fields in `LazyPath` (`generated` `path` and `cwd_relative`). The standard library part of the build system (the part run by the build runner) also does not really deal with "Packages". But I don't feel that strongly and I can change it if you still `package_file` would be better.